build.py

```diff
@@ -77,7 +77,7 @@
 CORE_BACKENDS = ['tensorrt', 'ensemble']
 NONCORE_BACKENDS = [
     'tensorflow1', 'tensorflow2', 'onnxruntime', 'python', 'dali', 'pytorch',
-    'openvino', 'fil', 'fastertransformer'
```

Comment:
It looks like  you are incorrectly reverting this earlier change: https://github.com/triton-inference-server/server/pull/3077

Reply:
Must have been a rebase issue. Thanks for pointing this out